silex_client/commands/move.py

```diff
@@ -53,18 +53,25 @@ async def __call__(
             # Execute the move in a different thread to not block the event loop
             def move():
                 # remove if dst already exist
-                # item : path of existing file/dir
-                end_path = os.path.join(dst, os.path.basename(item))
-                if os.path.isdir(end_path):
-                    shutil.rmtree(end_path)
 
-                if os.path.isfile(end_path):
-                    os.remove(end_path)
+                if os.path.isdir(dst):
+                    # clean tree
+                    shutil.rmtree(dst)
```

Comment:
Now all the shutil actions should use the wrapper in silex_client/silex_client/utils/thread.py
This is similar as wrapped_execute in maya and houdini, this will execute the shutil command in a new thread, so the event loop is not blocked. Here the probleme is that if the rmtree takes more that 10/20 seconds;, the websocket connection might close

Reply:
ok ty